src/client/dfs/dfs.c

```diff
@@ -1144,6 +1144,9 @@ dfs_mount(daos_handle_t poh, daos_handle_t coh, int flags, dfs_t **_dfs)
 	int			amode, obj_mode;
 	int			rc;
 
+        if (_dfs == NULL)
```

Comment:
(style)  code indent should use tabs where possible
(style)  suspect code indent for conditional statements (8, 10)